docs/website/content/v0.6/en/guides/local/firecracker.md

```diff
@@ -75,15 +75,25 @@ sudo cp cni-plugins-linux/{bridge,firewall,static} /opt/cni/bin
 
 ### Install tc-redirect-tap CNI plugin
 
-You should install CNI plugin from the Firecracker Go SDK repository [github.com/firecracker-microvm/firecracker-go-sdk/tree/master/cni](https://github.com/firecracker-microvm/firecracker-go-sdk/tree/master/cni)
+You should install CNI plugin from the `tc-redirect-tap` repository [github.com/awslabs/tc-redirect-tap](https://github.com/awslabs/tc-redirect-tap)
 
 ```bash
-go get -d github.com/firecracker-microvm/firecracker-go-sdk
-cd $GOPATH/src/github.com/firecracker-microvm/firecracker-go-sdk/cni
+go get -d github.com/awslabs/tc-redirect-tap/cmd/tc-redirect-tap
+cd $GOPATH/src/github.com/awslabs/tc-redirect-tap
 make all
 sudo cp tc-redirect-tap /opt/cni/bin
 ```
 
+> Note: if `$GOPATH` is not set, it defaults to `~/go`.
+
+## Build Talos kernel and initramfs
+
+Firecracker provisioner depends on Talos kernel and initramfs:
+
+```bash
+make kernel initramfs
+```
+
```

Comment:
An alternative is that they could also download these from a release.

Reply:
good point, let me add that, guide doesn't assume you have to be a developer, thanks!